modules/oc-mirror-about.adoc

```diff
@@ -20,6 +20,17 @@ When using the oc-mirror plugin, you specify which content to mirror in an image
 
 The first time you run the oc-mirror plugin, it populates your mirror registry with the required content to perform your disconnected cluster installation or update. In order for your disconnected cluster to continue receiving updates, you must keep your mirror registry updated. To update your mirror registry, you run the oc-mirror plugin using the same configuration as the first time you ran it. The oc-mirror plugin references the metadata from the storage backend and only downloads what has been released since the last time you ran the tool. This provides update paths for {product-title} and Operators and performs dependency resolution as required.
 
+[id="installation-oc-mirror-workflow_{context}"]
+== High level workflow
+The following steps outline the high-level workflow on how to use the oc-mirror plugin to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry by using one of the following methods:
```

Comment:
Better to change to ` Mirror the image set to the target mirror registry by using one of the following methods`